### PR TITLE
[palette_generator] Remove old and unused configuration file for gradlew

### PR DESCRIPTION
**Pull request by reidbaker**
Remove configuration file for gradlew that no longer exists. 
Pre-work for [#120119](https://github.com/flutter/flutter/issues/120119). 
gradlew removed in https://github.com/flutter/packages/pull/1338 but there are ~15 instances of gradlew in flutter/packages. 

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ x I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

### CODE REVIEW

**Review by gspencergoog — APPROVED** [review]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

File: packages/palette_generator/example/android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +0,0 @@
-#Tue Jan 07 08:46:39 PST 2020
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```
Comment (stuartmorgan) [inline packages/palette_generator/example/android/gradle/wrapper/gradle-wrapper.properties:6]: I thought this file was how we specified the gradle version to use; don't we have this in every Android example?

Comment (reidbaker) [reply packages/palette_generator/example/android/gradle/wrapper/gradle-wrapper.properties:6]: It does if there is a gradlew file and we call gradlew. I cant find the tooling that uses the locally defined gradle. When I run `gradle tasks` from `packages/packages/palette_genrator/example/android` I get the some warnings about migration to gradle 8 but tasks show correctly. My hope/expectation was that I would see unit tests or something fail in ci so I could fix/migrate this instead. 

Let me spend some time figuring out all the ways this code could be tested and make sure it passes locally.

Comment (stuartmorgan) [reply packages/palette_generator/example/android/gradle/wrapper/gradle-wrapper.properties:6]: `flutter build` creates the `gradlew` file. I believe `flutter build` and `flutter run` use it, and we also use it directly in the repo tooling here when running Android native unit tests. (That's why `config-only` will be useful; currently we have to make sure we run the unit tests after the test that everything builds)

**Pull request closed without merging**